stl/inc/xlocnum

```diff
@@ -1292,18 +1292,23 @@ protected:
         _OutIt _Dest, ios_base& _Iosbase, _Elem _Fill, double _Val) const { // put formatted double to _Dest
         string _Buf;
         char _Fmt[8];
-        bool _Isfixed         = (_Iosbase.flags() & ios_base::floatfield) == ios_base::fixed;
-        streamsize _Precision = _Iosbase.precision() <= 0 && !_Isfixed ? 6 : _Iosbase.precision(); // desired precision
-        size_t _Bufsize       = static_cast<size_t>(_Precision);
+        const bool _Isfixed = (_Iosbase.flags() & ios_base::floatfield) == ios_base::fixed;
+        const bool _Ishex   = (_Iosbase.flags() & ios_base::floatfield) == (ios_base::fixed | ios_base::scientific);
+        const streamsize _Precision =

```

Comment:
I get that this is a const initializer, but is there any way at all we could avoid the nested ternary, or at least parenthesize them?

Reply:
this is not the right line for this comment